src/engine/UIManager.cxx

```diff
@@ -1105,3 +1114,12 @@ void UIManager::changeFullScreenMode(UIElement *sender)
 
   MapFunctions::instance().refreshVisibleMap();
 }
+
+ImFont* UIManagerIm::buttonFont40 = nullptr;
+ImFont* UIManagerIm::buttonDefault = nullptr;
+void UIManagerIm::init() {
+  ImGuiIO& io = ImGui::GetIO();
+  buttonDefault = io.Fonts->AddFontDefault();
+  buttonFont40 = io.Fonts->AddFontFromFileTTF(Settings::instance().fontFileName.get().c_str(), 42.0f);
```

Comment:
why do you use a hardcoded font size here instead of using the default font size from settings?